src/css/mapbox-gl.css

```diff
@@ -761,6 +761,13 @@ a.mapboxgl-ctrl-logo.mapboxgl-compact {
     display: none;
 }
 
+.mapboxgl-accuracy-circle {
```

Comment:
We should include `user-location-dot` somewhere in this name so it's clear exactly what it relates too.

Reply:
updated: `mapboxgl-user-location-accuracy-circle`